File: ulyp-ui/src/main/java/com/ulyp/ui/renderers/RenderedNull.java
package com.ulyp.ui.renderers;

import com.ulyp.core.Type;
import com.ulyp.ui.RenderSettings;
import javafx.scene.text.Text;

public class RenderedNull extends RenderedObject {

    RenderedNull(RenderSettings renderSettings) {
        super(Type.unknown());

        super.getChildren().add(new Text("null"));
    }
}

File: ulyp-ui/src/main/kotlin/com/ulyp/ui/renderers/RenderedNull.kt
package com.ulyp.ui.renderers

import com.ulyp.core.Type
import com.ulyp.ui.RenderSettings
import com.ulyp.ui.util.CssClass
import com.ulyp.ui.util.StyledText
import javafx.scene.text.Text

class RenderedNull internal constructor(renderSettings: RenderSettings?) : RenderedObject(Type.unknown()) {

    init {
        super.getChildren().add(StyledText.of("null", CssClass.CALL_TREE_NULL))
    }
}